### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by TheUltra12** [community/community#132602] [Repositories]
### Select Topic Area

Question

### Body

Will Geode be added to iOS? If yes, when? I really want to play one of your mods “Globed”!

**Comment by cadzchua** [comment]
nope, ios limitations

**Reply by TheUltra12** [reply]
What does that mean?

**Comment by queenofcorgis** [comment]
Hi @TheUltra12 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Geode. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
